apps/coscrad-frontend/src/components/resources/photographs/photograph-index.presenter.tsx

```diff
@@ -1,21 +1,34 @@
 import { AggregateType, IPhotographViewModel } from '@coscrad/api-interfaces';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../../configurable-front-matter/configurable-content-provider';
 import { PhotographIndexState } from '../../../store/slices/resources/photographs/types';
 import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { Matchers } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/filter-table-data';
 import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { doesSomeMultilingualTextItemInclude } from '../utils/query-matchers';
 import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+import { renderMultilingualTextCell } from '../utils/render-multilingual-text-cell';
 import { renderPhotographThumbnailLinkCell } from '../utils/render-photograph-thumbnail-link-cell';
 
 export const PhotographIndexPresenter = ({ entities: photographs }: PhotographIndexState) => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
     const headingLabels: HeadingLabel<IPhotographViewModel>[] = [
         { propertyKey: 'id', headingLabel: 'Link' },
-        { propertyKey: 'imageUrl', headingLabel: 'Image URL' },
-        { propertyKey: 'photographer', headingLabel: 'Photographer' },
+        { propertyKey: 'imageUrl', headingLabel: 'Image' },
```

Comment:
At some point, I could imagine pulling a thumbnail from the media server. We'll need to add support for downsampling first.